clang/bindings/python/tests/cindex/test_file.py

```diff
@@ -1,12 +1,13 @@
 import os
 
-from clang.cindex import Config, File, Index
+from clang.cindex import Config, File, Index, TranslationUnit
 
 if "CLANG_LIBRARY_PATH" in os.environ:
     Config.set_library_path(os.environ["CLANG_LIBRARY_PATH"])
 
 import unittest
 
+kInputsDir = os.path.join(os.path.dirname(__file__), "INPUTS")
```

Comment:
[PEP 8][https://peps.python.org/pep-0008/#global-variable-names], which we [follow](https://llvm.org/docs/CodingStandards.html#python-version-and-source-code-formatting), says that global variable names should be named in the same way as functions, which are `snake_case` in this file.

Reply:
Oh wow, didn't know global variables should be snake case. I copied the naming straight from one of the other files and we have a couple more places where this is the case, so I made #132378 first and added you as reviewe. Will rebase this PR once the other one is merged.